mailbox/api/src/main/java/org/apache/james/mailbox/MailboxListener.java

```diff
@@ -63,10 +64,12 @@
     abstract class Event implements Serializable {
         private final MailboxSession session;
         private final MailboxPath path;
+        private final MailboxId mailboxId;
 
-        public Event(MailboxSession session, MailboxPath path) {
+        public Event(MailboxSession session, MailboxPath path, MailboxId mailboxId) {
```

Comment:
We said in Sprint planning **we forgot aboit the mailboxId stuff in order to deliver functionnality quickly**.